Contributing.md

```diff
@@ -1,79 +1,81 @@
-- Add Screenshots to help us know what this Script is all about.
+- Take a look at the existing [Issues](https://github.com/EddieHubCommunity/LinkFree/issues) or [create a new issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose)!
+- [Fork the Repo](https://github.com/EddieHubCommunity/LinkFree/network/members), create a branch for any issue that you are working on and commit your work.
+- Create a **[Pull Request](https://github.com/EddieHubCommunity/LinkFree/compare)** (*PR*) which will be promptly reviewed and given suggestions for improvement by the community.
+- Add screenshots or screen captures to your Pull Request to help us understand the effects of the changes that are included in your commits.
 
 
-## ⭐HOW TO MAKE A PULL REQUEST:
+## ⭐ HOW TO MAKE A PULL REQUEST:
 
-**1.** Fork [this](https://github.com/EddieHubCommunity/LinkFree) repository.
-Click on the <a href="https://github.com/EddieHubCommunity/LinkFree/network/members"><img src="https://img.icons8.com/ios/24/32CD32/code-fork.png"></a> symbol at the top right corner.
+**1.** Start by making a fork the [**LinkFree**](https://github.com/EddieHubCommunity/LinkFree) repository. Click on the <a href="https://github.com/EddieHubCommunity/LinkFree/network/members"><img src="https://img.icons8.com/ios/24/32CD32/code-fork.png"></a> symbol at the top right corner.
 
-**2.** Clone the forked repository.
+**2.** Clone your new fork of the repository:
 
 ```bash
 git clone https://github.com/<your-github-username>/LinkFree
 ```
 
-**3.** Navigate to the project directory.
+**3.** Navigate to the new project directory:
 
 ```bash
 cd LinkFree
 ```
+
 **4.** Create a new branch:
 ```bash
 git checkout -b YourBranchName
 ```
 
-**5.** Make changes in source code.
+**5.** Make your changes to the source code.
 
-**6.** Stage your changes and commit
+**6.** Stage your changes and commit:
 
 ```bash
 git add .
 ```
+
 ```bash
 git commit -m "<your_commit_message>"
 ```
 
-**7.** Push your local commits to the remote repo.
+**7.** Push your local commits to the remote repository:
 
 ```bash
 git push origin YourBranchName
 ```
 
-**8.** Create a [PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) !
+**8.** Create a [Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)!
+
+**9.** **Congratulations!** You've made your first contribution to [**LinkFree**](https://github.com/EddieHubCommunity/LinkFree/graphs/contributors)! 🙌🏼
 
-**9.** **Congratulations!** Sit and relax, you've made your contribution to [LinkFree](https://github.com/EddieHubCommunity/LinkFree) project.
+***:trophy: After this, project leaders and mentors will review the PR and will merge it if it helps move the LinkFree project forward. Otherwise it will be given constructive feedback and suggestions for the changes needed add the PR to the codebase.***

```

Comment:
```suggestion
***:trophy: After this, the maintainers will review the PR and will merge it if it helps move the LinkFree project forward. Otherwise, it will be given constructive feedback and suggestions for the changes needed to add the PR to the codebase.***
```